client/src/pages/Review/Review.tsx

```diff
@@ -2,122 +2,80 @@ import { useState } from "react";
 import { useParams, Link } from "react-router-dom";
 import Button from "../../components/Button";
 import { classnames } from "../../Utilities";
+import "./Review.css";
```

Comment:
Use modularized stylesheet.

```suggestion
import styles from "./Review.module.css";
```